trc.py

```diff
@@ -98,6 +98,7 @@ def chech_root():
 
 def secure_server():
     print_info("Securing server...")
+    run_command("apt-get install ufw")
```

Comment:
This package is installed by default on ubuntu 16.04. Any reason adding it explicitly?

Reply:
It is missing in multiple default Ubuntu 16.04 vps providers (tested 3 andere none had ufw)